projects/eventonica/eventonica-part6-postgres.md

```diff
@@ -94,9 +95,11 @@ TL;DR - they are taking their in-memory backend data objects from Part 5 and usi
 
 - README should contain instructions on how to load the testing database schema (likely with data)
 - A big part of reviewing this is checking it out and making sure it works
-- I've been using a new [beta GitHub CLI](https://cli.github.com/) that can quickly checkout a PR
 - README should also mention how to run any tests
-- SQL commands should be in the EventRecommender "DAO" object, not in the Express app route handlers
-- If the code is all stuffed into the handlers, send your preferred explanatory link about the concept of system layers
+- SQL commands should be in the model objects, not in the Express app route handlers
+  - If the code is all stuffed into the handlers, send your preferred explanatory link about the concept of system layers. Our curriculum doesn't currently have a lesson for it.
 - If there are no unit tests or API tests, flag that as an area of possible improvement
```

Comment:
@SarahCawley, any thoughts on how to handle this?

Reply:
IIRC this was instructions to the volunteer assignment graders actually. May or may not be a feature request.

Reply:
@vegetabill, thanks for clarifying!